invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/index.js

```diff
@@ -432,6 +550,71 @@ class CommunityProfileForm extends Component {
                     allowAdditions
                     search={(filteredOptions, searchQuery) => filteredOptions}
                   />
+                  <FundingField
+                    fieldPath="metadata.funding"
+                    searchConfig={{
+                      searchApi: {
+                        axios: {
+                          headers: {
+                            //  FIXME use for internationalisation
+                            //  Accept: "application/vnd.inveniordm.v1+json"
+                            Accept: "application/json",
+                          },
+                          url: "https://127.0.0.1:5000/api/awards",
+                          withCredentials: false,
+                        },
+                      },
+                      initialQueryState: {
+                        sortBy: "bestmatch",
+                        sortOrder: "asc",
+                        layout: "list",
+                        page: 1,
+                        size: 5,
+                      },
+                    }}
+                    label="Awards"
+                    labelIcon="money bill alternate outline"
+                    deserializeAward={(award) => {
+                      return {
+                        title: award.title.en ?? award.title,
+                        pid: award.pid,
+                        number: award.number,
+                        funder: award.funder ?? '',
+                        id: award.id,
+                        ...(award.identifiers && { identifiers: award.identifiers }),
+                        ...(award.acronym && { acronym: award.acronym })
+                      }
+                    }}
+                    deserializeFunder={(funder) => {
+                      return {
+                        id: funder.id,
+                        name: funder.name,
+                        ...(funder.pid && { pid: funder.pid }),
+                        ...(funder.country && { country: funder.country }),
+                        ...(funder.identifiers && { identifiers: funder.identifiers })
+                      }
+                    }
+                    }
+                    computeFundingContents={(funding) => {
+                      let headerContent, descriptionContent = '';
+                      let awardOrFunder = 'award';
+                      if (funding.award) {
+                        headerContent = funding.award.title;
+                      }
+
+                      if (funding.funder) {
+                        const funderName = funding.funder?.name ?? funding.funder?.title?.en ?? funding.funder?.id ?? '';
```

Comment:
Shouldn't the i18n title come first? Is the funder a person or organization type? If so, isn't it better to check explicitly what type of funder you have and be stricter with where to check for the name?